### ISSUE TITLE
can not use smith

### ISSUE THREAD

**Issue by Nankarina** [chartjs/chartjs-chart-smith#78]
`  <script src="https://cdn.jsdelivr.net/npm/chart.js@4.3.0/dist/chart.umd.min.js"></script>`

"smith" is not a registered controller.

**Comment by etimberg** [comment]
@Nankarina are you including both scripts?

I haven't tested the extension with chart.js v4, so that might be part of the problem

**Comment by Nankarina** [comment]
> @Nankarina are you including both scripts?
> 
> I haven't tested the extension with chart.js v4, so that might be part of the problem

yes I including both of them but can not work . I resolved this problem in the low script version
